src/libraries/System.Runtime/tests/System/TimeZoneInfoTests.cs

```diff
@@ -160,8 +177,9 @@ public static void LibyaTimeZone()
             Assert.True(libyaLocalTime.Equals(expectResult), string.Format("Expected {0} and got {1}", expectResult, libyaLocalTime));
         }
 
-        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsWindows))]
-        public static void TestYukunTZ()
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
Why the change from `ConditionalFact`?

Reply:
Just for consistency.  There were already several other uses of `[PlatformSpecific]`, and that's the only condition being checked.